apps/televisit-demo/backend/messagemoderator/index.js

```diff
@@ -0,0 +1,106 @@
+const AWS = require("aws-sdk");
+AWS.config.update({ region: process.env.AWS_REGION });
+var translate = new AWS.Translate();
+var comprehend = new AWS.Comprehend();
+const lex = new AWS.LexRuntime();
+
+const {
+  sendChannelMessage,
+  updateChannelMessage,
+  listChannelMessages
+} = require("./ChimeMessagingAPI");
+
+console.log('Loading function');
+
+const { DOCUMENT_CLASSIFIER_ENDPOINT, CHIME_APP_INSTANCE_ARN, CHIME_APP_INSTANCE_ADMIN_ROLE_ARN } = process.env; 
+
+exports.handler = function(event, context) {
+
+
+  event.Records.forEach(async (record) => {
+    
+    // Kinesis data is base64 encoded so decode here
+    var payload = Buffer.from(record.kinesis.data, 'base64').toString();
+    console.log('Decoded payload:', payload);
+    
+    //Parsing payload as a JSON message payload
+    var jsonMessage = JSON.parse(payload);
+    var senderId = jsonMessage.Payload.Sender.Arn.substring(jsonMessage.Payload.Sender.Arn.lastIndexOf("/")+1);
+    var channelArn = jsonMessage.Payload.ChannelArn;
+    var messageId = jsonMessage.Payload.MessageId;
+    
+    
+    console.log('Message content:', jsonMessage.Payload.Content);
+    if (jsonMessage.Payload.Content.startsWith("/")){

```

Comment:
It's hard to understand what this `if` block does, e.g., what is the content starting with `/`?

Recommend adding comments.